Libraries/LibGfx/Font/FontVariationSettings.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2025, Norbiros <me@norbiros.dev>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/HashMap.h>
+#include <AK/QuickSort.h>
+#include <AK/Vector.h>
+#include <LibGfx/FourCC.h>
+
+#pragma once
+
+namespace Gfx {
+
+struct FontVariationAxis {
+    FourCC tag;
+    float value;
+
+    FontVariationAxis(FourCC t, float v)
+        : tag(t)
+        , value(v)
+    {
+    }
+
+    bool operator==(FontVariationAxis const& other) const
+    {
+        return tag == other.tag && value == other.value;
+    }
+};
+
+// FIXME: Support other named axes like 'slnt', 'ital', 'opsz', 'GRAD', etc.
+struct FontVariationSettings {
+    HashMap<FourCC, float> axes;
```

Comment:
if we know all possible axes in advance, let's just preallocate a vector large enough to hold all of them instead of creating a HashMap, which would only ever hold at most two values.